sdk/python/kfp/components/_airflow_op.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+__all__ = [
+    'create_component_from_airflow_op',
+]
+
+
+from typing import List
+
+from ._python_op import _func_to_component_spec, _create_task_factory_from_component_spec
+
+
+_default_airflow_base_image = 'apache/airflow@sha256:7f60cbef6bf92b1f3a5b4e46044911ced39736a8c3858284d3c5a961b3ba8735'
+
+
+def create_component_from_airflow_op(op_class, base_image=_default_airflow_base_image, result_output_name='Result', variable_output_names=None, xcom_output_names=None, modules_to_capture: List[str] = None):
```

Comment:
Could you add an example on how to use it?

Reply:
The description of this PR has an example, but I can add one in the code.

Reply:
A basic sample will also increase the visibility of the feature. 
Do you know if the code is ready to ship? any known limitation?

Reply:
> Do you know if the code is ready to ship? any known limitation?

This PR depended on https://github.com/kubeflow/pipelines/pull/1435 https://github.com/kubeflow/pipelines/pull/1511 and is also affected by https://github.com/kubeflow/pipelines/issues/1485, but the biggest issue for now is https://github.com/kubeflow/pipelines/issues/1507 .

The new notebooks are running python 3.6. Fortunately, `airflow` images also run `3.6`, so this PR can probably move forward soon. (Bad news is that `tensorflow` images are `3.5`)

Reply:
sg. thanks for the info.